config/oidc/config.proto

```diff
@@ -172,3 +172,120 @@ message OIDCConfig {
     // Optional.
     RedisConfig redis_session_store_config = 16;
 }
+
+// This OIDC config doesn't need to be validate. It is introduced to use with `default_oidc_config` and `oidc_override`
+// LooseOIDCConfig will be merged to construct OIDC Config and validated.
+message LooseOIDCConfig {
```

Comment:
Not fan of this kind of copy proto. You should be able to control where PGV kicks in in the code.